app/domain/authentication/authn_jwt/validate_and_decode/fetch_audience_value.rb

```diff
@@ -0,0 +1,79 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module ValidateAndDecode
+      # Fetch and validate the audience from the JWT authenticator policy
+      FetchAudienceValue = CommandClass.new(
+        dependencies: {
+          resource_class: ::Resource,
+          fetch_required_secrets: ::Conjur::FetchRequiredSecrets.new,
+          logger: Rails.logger
+        },
+        inputs: %i[authentication_parameters]
+      ) do
+        def call
```

Comment:
add info message of the final audience value

Reply:
Done